docs/Hero.md

```diff
@@ -53,6 +53,7 @@ Parent element (`<pxb-hero>`) attributes:
 | iconSize | The size of the primary icon (10-48)   | `number`             | no       | 'normal' (36) |
 | label    | The text shown below the Channel Value | `string`             | yes      |               |
 | units    | Text to show after the value           | `string`             | no       |               |
+| unitSpace  | Show/Hide spacing between the value and units  | `'auto'` \| `'hide'` \| `'show'` | yes      | 'auto'  |
```

Comment:
I believe this should be required: _no_ (it has a default value and thus should be optional).